### PR TITLE
chore: increase coverage target to 85% and add tests to reach it

### PR DESCRIPTION
**Pull request by is0692vs**
## Summary
This PR increases the project-wide Codecov coverage target from 80% to 85% and adds numerous test cases across both `api` and `web` apps to meet this new goal.

### Changes
- **codecov.yml**: Updated project target to 85% and patch target to 75%.
- **apps/api**:
  - Improved coverage for `auth`, `orgs`, `papers`, and `users` routes.
  - Added tests for edge cases like unauthorized access, validation errors, and complex invite flows.
  - Fixed a minor bug in `PATCH /api/users/me` where it wouldn't return 400 on empty updates.
- **apps/web**:
  - Increased coverage for `SettingsPage`, `UploadPage`, `InvitesPage`, and `OrgPage`.
  - Refined test mocks to handle `useEffect` cycles and async data loading more robustly.
  - Fixed `lib/api` coverage by adding SSR branch tests.

### Result
- `apps/api`: ~85.8% coverage
- `apps/web`: ~84.2% coverage
- **Total Average: ~85.0%** (Goal Reached)

### CODE REVIEW

File: apps/api/src/routes/__tests__/papers.test.ts [hunk 1]
```diff
@@ -841,4 +848,351 @@ describe("papers routes", () => {
...
+                },
+                body: JSON.stringify({ unknownField: "val" }),
+            },
+            env as any
+        );
+        expect(res13.status).toBe(400);
+        expect(await res13.json()).toEqual({ error: "No valid fields to update" });
+    });
+
+    it("POST /api/papers rejects invalid metadata", async () => {        const app = await createTestApp();
```
[the remark this answers is not in the document]
Comment (is0692vs) [reply apps/api/src/routes/__tests__/papers.test.ts]: 対応しました: invalid metadata テストの関数宣言直後の改行を修正しました。検証: npm run test, npm run lint, npm run typecheck.

File: apps/web/src/app/invites/__tests__/page.test.tsx [hunk 2]
```diff
@@ -100,4 +105,92 @@ describe("InvitesPage", () => {
...
+                createdAt: "2026-03-01T00:00:00Z",
+              },
+            ],
+          }),
+          { status: 200 },
+        );
+      }
+      if (url === "/api/invites/invite-2" && init?.method === "PATCH") {
+        return new Response("{}", { status: 200 });
+      }
```
[the remark this answers is not in the document]
Comment (is0692vs) [reply apps/web/src/app/invites/__tests__/page.test.tsx]: 対応しました: decline 側の PATCH モックでも action が decline であることを検証するように揃えました。検証: npm run test, npm run lint, npm run typecheck.
